src/diagnostics/define_diagnostics.jl

```diff
@@ -179,130 +181,276 @@ function define_diagnostics!(land_model)
 
     ###### SoilCanopyModel ######
 
+    ## stored in p (diagnostics variables stored in the cache) ##
+
+    ## Canopy Module ##
+	
+	### Canopy - Solar Induced Fluorescence 
+	    # Solar Induced Fluorescence
     add_diagnostic_variable!(
-        short_name = "slw",
-        long_name = "Soil Liquid Water",
-        standard_name = "soil_liquid_water",
-        units = "m^3 m^-3",
-        comments = "Soil moisture, the liquid water volume per soil volume. This liquid water is located in the soil pores.",
+        short_name = "sif",
+        long_name = "Solar Induced Fluorescence",
+        standard_name = "solar_induced_fluorescence",
+        units = "W m^-2",
+        comments = "The fluorescence of leaves induced by solar radiation. This quantity is correlated with photosynthesis activity.",
         compute! = (out, Y, p, t) ->
-            compute_soil_water_liquid!(out, Y, p, t, land_model),
+            compute_solar_induced_fluorescence!(out, Y, p, t, land_model),
     )
 
+    ### Canopy - Autotrophic respiration
+        # Autotrophic respiration
     add_diagnostic_variable!(
-        short_name = "infil",
-        long_name = "Infiltration",
-        standard_name = "infiltration",
-        units = "m s^-1", # double check
-        comments = "The flux of liquid water into the soil.",
+        short_name = "ra",
+        long_name = "Autotrophic Respiration",
+        standard_name = "autotrophic_respiration",
+        units = "mol CO2 m^-2 s^-1", # note throughout: make sure what we express as kg or mol, CO2 or C
+        comments = "The canopy autotrophic respiration, the sum of leaves, stems and roots respiration.",
         compute! = (out, Y, p, t) ->
-            compute_infiltration!(out, Y, p, t, land_model),
+            compute_autotrophic_respiration!(out, Y, p, t, land_model),
     )
 
+    ### Canopy - Conductance
+        # Stomatal conductance
     add_diagnostic_variable!(
-        short_name = "scd",
-        long_name = "Soil CO2 Diffusivity",
-        standard_name = "soil_co2_diffusivity",
-        units = "", # need to add
-        comments = "The diffusivity of CO2 in the porous phase of the soil. Depends on soil texture, moisture, and temperature.",
+        short_name = "gs",
+        long_name = "Stomatal Conductance",
+        standard_name = "stomatal_conductance",
+        units = "m s^-1",
```

Comment:
mol/m^2/s
https://github.com/CliMA/ClimaLand.jl/blob/main/src/standalone/Vegetation/canopy_parameterizations.jl#L908

Reply:
amended to `mol H2O m^-2 s^-1`